docs/spec/spec.md

```diff
@@ -3934,4 +3934,51 @@ service /graphql on new graphql:Listener(9090) {
 }
 ```
 
-In this example, caching is enabled at the operation level. Therefore, the field `name` and `type` will be cached. Since the field-level cache configuration overrides the parent cache configurations, the field `version` will not be cached. When updating the name with a mutation, the cached values become invalid. Hence, the `invalidate` function can be used to ivalidate the existing cache values.
+In this example, caching is enabled at the operation level. Therefore, the field `name` and `type` will be cached. Since the field-level cache configuration overrides the parent cache configurations, the field `version` will not be cached. When updating the name with a mutation, the cached values become invalid. Hence, the `invalidate` function can be used to invalidate the existing cache values.
+
+### 10.8 Observability
```

Comment:
Any reason for not updating the `contents` sections?

Reply:
Missed this, will update.